core/src/main/scala/org/broadinstitute/dsde/rawls/methods/MethodConfigurationUtils.scala

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.dsde.rawls.methods
+
+import akka.http.scaladsl.model.StatusCodes
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.dataaccess.MethodRepoDAO
+import org.broadinstitute.dsde.rawls.jobexec.MethodConfigResolver
+import org.broadinstitute.dsde.rawls.model.{ErrorReport, MethodConfiguration, RawlsRequestContext}
+import org.broadinstitute.dsde.workbench.util.FutureSupport.toFutureTry
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+object MethodConfigurationUtils {
```

Comment:
A `trait` could also be nice here but I don't see it as a compelling need to change.